src/test/testWebpackSourceMaps.ts

```diff
@@ -26,43 +27,41 @@ describe('Webpack sourcemaps: The debugger', function() {
 		}
 	});
 
-	for (let devtool of [
-		'cheap-eval-source-map', 'cheap-source-map', 'cheap-module-eval-source-map', 'inline-source-map',
-		'cheap-module-source-map' , 'eval-source-map' , 'source-map' , 'nosources-source-map'
-	]) {
+	for (const devtool of [
+		'source-map', 'inline-source-map', 'nosources-source-map', 'inline-nosources-source-map',
+		'eval-source-map', 'eval-cheap-source-map', 'eval-cheap-module-source-map',
+		'eval-nosources-source-map', 'eval-nosources-cheap-source-map', 'eval-nosources-cheap-module-source-map',
+	] satisfies Devtool[]) {
 
 		const description = `should map webpack-bundled modules with devtool "${devtool}" to their original sources`;
-
-		// disable tests with webpack devtools that are known to be broken (webpack bug #5491)
-		if (devtool.indexOf('eval') < 0) {
-			it.skip(description);
-			continue;
-		}
+		const isEvalSourcemap = devtool.indexOf('eval') >= 0;
+		const isCheapSourcemap = devtool.indexOf('cheap') >= 0;
```

Comment:
Maybe we could use `startsWith` and `includes` ?
```suggestion
		const isEvalSourcemap = devtool.startsWith('eval');
		const isCheapSourcemap = devtool.includes('cheap');
```